### PR TITLE
0.2.7 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR add release notes for 0.2.7 including highlighting deprecations and api changes

### CODE REVIEW

File: docs/release/release_0_2_7.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Iso-surface threshold slider (#712)
+- Add play button to gui (#726)
+- Make layers list dockable (#727)
+- Add Zenodo badge to documentation (#743)
+- Add a dock icon (#744)
+- Show splash screen for cli launch (#745)
+- Add benchmarks for setting .data in Image layers (#747)
+- Refactor layer tests to be more parametrized (#723)
+- Support opening labels layers via directly from path (#748)
+- Simplify keybindings info display and make dockable (#749)
```
Comment (tlambert03) [inline docs/release/release_0_2_7.rst]:
```suggestion
- Simplify keybindings info display (#749)
```
we ended up not making this dockable right?

File: docs/release/release_0_2_7.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+********
+- Fix StringEnum setting and errors (#757)
+- scale argument now accepts array-like input (#765)
+- fix set_fps type to float (#767)
+- Add shutdown method to QtViewer that closes all resources (#769)
+- Change language around windows support in readme (#779)
+- Revert #784 console shutdown conditionals (#796)
+- Fix window raise & inactive menubar conflict (#795)
+- Change documentation on qt.py folder location (#783)
+- Updating qt_console with better resource management.  (#784)
```
Comment (tlambert03) [inline docs/release/release_0_2_7.rst]:
```suggestion
- Updating qt_console with better resource management (#784)
```
(format consistency)

**Review by tlambert03 — APPROVED** [review]
🚀

File: docs/release/release_0_2_7.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst]: correct, good catch. fixed

File: docs/release/release_0_2_7.rst (hunk 2 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst]: fixed

**Comment by sofroniewn** [comment]
I have made the minor edits requested by @tlambert03 - if this looks good to you @jni I will merge and start making the release candidate

File: docs/release/release_0_2_7.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
+Announcement: napari 0.2.7
+==========================
+
+We're happy to announce the release of napari v0.2.7!
```
Comment (jni) [inline docs/release/release_0_2_7.rst]: Has this v always been here? I don't love it, compared to just 0.2.7, as above.

File: docs/release/release_0_2_7.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-base
```
Comment (jni) [inline docs/release/release_0_2_7.rst]:
```suggestion
images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
```

File: docs/release/release_0_2_7.rst (hunk 3 above)
Comment (jni) [reply docs/release/release_0_2_7.rst]:
```suggestion
We're happy to announce the release of napari 0.2.7!
```

File: docs/release/release_0_2_7.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+- Move Layer utility functions to /napari/layers/layer_utils.py (#778)
+- Refactor util.misc (#781)
+
+Deprecations
+************
+- drop ndim keyword from labels layer (#773)
+
+
+7 authors added to this release [alphabetical by first name or login]
+---------------------------------------------------------------------
```
Comment (jni) [inline docs/release/release_0_2_7.rst:68]: Hmm. I get that reviews are useful but I need to get back into coding. 😂

File: docs/release/release_0_2_7.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+***********
+- Rename util to utils across repo (#808)
+- Move Labels utility functions to labels_util.py (#770)
+- Move Image layer utility functions to image_utils.py (#775)
+- Move Layer utility functions to /napari/layers/layer_utils.py (#778)
+- Refactor util.misc (#781)
+
+Deprecations
+************
+- drop ndim keyword from labels layer (#773)
```
Comment (jni) [inline docs/release/release_0_2_7.rst]: This is an API change, not a deprecation, I believe? Deprecation -> warning that something will be removed in the future.

File: docs/release/release_0_2_7.rst [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+- Add shutdown method to QtViewer that closes all resources (#769)
+- Change language around windows support in readme (#779)
+- Revert #784 console shutdown conditionals (#796)
+- Fix window raise & inactive menubar conflict (#795)
+- Change documentation on qt.py folder location (#783)
+- Updating qt_console with better resource management (#784)
+- Respect vispy max texture limits (#788)
+- Fix (minor) deprecation warnings (#800)
+- Fix FPS spin box on Qt < 5.12 (#803)
+- Bumpy vispy 0.6.4 (#807)
```
Comment (jni) [inline docs/release/release_0_2_7.rst]:
```suggestion
- Bumpy vispy dependency to 0.6.4 (#807)
```

File: docs/release/release_0_2_7.rst [hunk 8]
```diff
@@ -0,0 +1,81 @@
+Announcement: napari 0.2.7
+==========================
+
+We're happy to announce the release of napari v0.2.7!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-base
+rendering), and the scientific Python stack (numpy, scipy).
+
+
```
Comment (jni) [inline docs/release/release_0_2_7.rst:10]: I would add a highlights section here. Here's the PRs that I think would be particularly exciting from a user's perspective:

- threshold slider for much improved isosurface rendering
- dockable widgets (!)
- slice information on sliders
- dramatically improved performance with many invisible layers

**Review by jni — COMMENTED** [review]
@sofroniewn I've left some comments that would be good to address before merging. I'll leave this as a comment though so things aren't blocked while I'm asleep!

File: docs/release/release_0_2_7.rst (hunk 3 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst]: fixed

File: docs/release/release_0_2_7.rst (hunk 4 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst]: fixed

File: docs/release/release_0_2_7.rst (hunk 6 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst]: ah yes, my mistake

File: docs/release/release_0_2_7.rst (hunk 7 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst]: done

File: docs/release/release_0_2_7.rst (hunk 8 above)
Comment (sofroniewn) [reply docs/release/release_0_2_7.rst:10]: added a highlight section with these and the play button!

**Comment by sofroniewn** [comment]
I have made the changes requested by @jni and the tests are passing so I will merge

**Pull request merged**